### ISSUE TITLE
Add information about Proguard rules

### ISSUE THREAD

**Issue by Wojtek1234** [EdwardvanRaak/MaterialBarcodeScanner#35]
Hey, you should provide proguard rules in Readme.md, EventBus does not do that on their main page, so there is a confusion after obfuscation wtf is going on, why nothing works

here there are
`-keepattributes *Annotation*
-keepclassmembers class ** {
    @org.greenrobot.eventbus.Subscribe <methods>;
}
-keep enum org.greenrobot.eventbus.ThreadMode { *; }
 
# Only required if you use AsyncExecutor
-keepclassmembers class * extends org.greenrobot.eventbus.util.ThrowableFailureEvent {
    <init>(java.lang.Throwable);
}`

**Comment by AllanWang** [comment]
I agree. And keep in mind you can also add proguard through your own library with `consumerProguardFiles 'progress-proguard.txt'` under `defaultConfig`
